extension.json

```diff
@@ -107,6 +107,14 @@ set +x
 set +e 
 export INIT_START_TIME=$(date +\"%s\")
 
+#############
+# Colors    #
+#############
+export green=\'\e[0\;32m\'
+export red=\'\e[0\;31m\'
+export label_color=\'\e[0\;33m\'
+export no_color=\'\e[0m\'
```

Comment:
I _think_ all of this escaping should work properly when actually run in the context of the pipeline, @eLobeto is there a way I could test this branch out on dev?